zenfs.mk

```diff
@@ -1,10 +1,12 @@
 zenfs_SOURCES = fs/fs_zenfs.cc fs/zbd_zenfs.cc fs/io_zenfs.cc
 zenfs_HEADERS = fs/fs_zenfs.h fs/zbd_zenfs.h fs/io_zenfs.h fs/version.h fs/metrics.h fs/snapshot.h
-zenfs_LDFLAGS = -lzbd -u zenfs_filesystem_reg
+zenfs_LDFLAGS = -u zenfs_filesystem_reg
```

Comment:
Do we not need to link the zbd library anymore ?

Reply:
It's handled in line 12 by pkg-config.

Reply:
ok